pkg/config/cos.go

```diff
@@ -434,6 +442,23 @@ func updateBond(stage *yipSchema.Stage, name string, network *Network) error {
 	return nil
 }
 
+func updateVLAN(stage *yipSchema.Stage, name string, network *Network) error {
+	n := networkWithName{Name: name, Network: *network}
```

Comment:
NIT: anonymous struct might be better for local use?

```suggestion
	n := struct {
		Name    string
		Network Network
	}{
		Name: name,
		Network: *network,
	}
```

Reply:
Updated it. Thank you.